acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
So if facter.rb fails to load the libfacter.jar (if its jRuby) or fails to load the libfacter.so, then in both cases we raise an error which you would see in the output from ruby.

    begin
      require 'facter.jar'
    rescue LoadError
      raise LoadError.new('libfacter was not built with JRuby support.')
    end

      require "#{facter_dir}/lib/libfacter.so"
    rescue LoadError
      raise LoadError.new('libfacter was not found. Please make sure it was installed to the expected location.')
    end